packages/web3-wrapper/src/web3_wrapper.ts

```diff
@@ -171,15 +168,22 @@ export class Web3Wrapper {
      * Retrieve the Web3 provider
      * @return  Web3 provider instance
      */
-    public getProvider(): Provider {
+    public getProvider(): SupportedProvider {
+        return this._supportedProvider;
+    }
+    /**
+     * Retrieve the Web3 provider
+     * @return  Web3 provider instance
+     */
+    public getZeroExProvider(): ZeroExProvider {
```

Comment:
How often do we use this? Seems only to be used in one test

Reply:
Removed.